bigbluebutton-html5/imports/ui/components/plugins-engine/loader/manager.tsx

```diff
@@ -40,8 +40,8 @@ const PluginLoaderManager = (props: PluginLoaderManagerProps) => {
     script.src = plugin.url;
     script.setAttribute('uuid', div.id);
     script.setAttribute('pluginName', plugin.name);
-    if (plugin.checksum) {
-      script.setAttribute('integrity', plugin.checksum);
+    if (plugin.javascriptEntrypointChecksum) {
+      script.setAttribute('integrity', plugin.javascriptEntrypointChecksum);
```

Comment:
Since this field will be directly used as the `integrity` attribute in the `<script>` tag, I suggest renaming it to `javascriptEntrypointIntegrity`. The term "checksum" usually refers to just the hash, while "integrity" is slightly different, as explained here: https://www.w3schools.com/tags/att_script_integrity.asp.